# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/pandas/hotfix-gitversion.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- pandas-2.2.3/generate_version.py.vanilla	2025-03-11 00:18:03.067272414 +0100
+++ pandas-2.2.3/generate_version.py	2025-03-11 00:19:28.664414110 +0100
@@ -5,23 +5,13 @@
 import os
 import sys
 
-import versioneer
-
 sys.path.insert(0, "")
 
 
 def write_version_info(path) -> None:
-    version = None
-    git_version = None
-
-    try:
-        import _version_meson
+    version = os.environ.get("ver")
+    git_version = os.environ.get("ver")
 
-        version = _version_meson.__version__
-        git_version = _version_meson.__git_version__
-    except ImportError:
-        version = versioneer.get_version()
-        git_version = versioneer.get_versions()["full-revisionid"]
     if os.environ.get("MESON_DIST_ROOT"):
         path = os.path.join(os.environ.get("MESON_DIST_ROOT"), path)
     with open(path, "w", encoding="utf-8") as file:
